common/build/eslint-config-fluid/.eslintrc.js

```diff
@@ -5,15 +5,7 @@
 
 module.exports = {
     "extends": [
-        // "@rushstack/eslint-config/profile/web-app",
-        "eslint:recommended",
-        "plugin:eslint-comments/recommended",
-        "plugin:import/errors",
-        "plugin:import/warnings",
-        "plugin:import/typescript",
-        // "@rushstack/eslint-config/mixins/tsdoc",
-        // "plugin:unicorn/recommended",
-        "plugin:editorconfig/all",
+        "./minimal"
```

Comment:
Why?  What does this .eslintrc.js file do?

Reply:
It's the config that applies to the local ts files -- but I am explicitly passing a config when I call eslint (see the package.json scripts) so this file can just be deleted. That should help reduce confusion.